### PR TITLE
Email activation

### PR DESCRIPTION
**Pull request by madhalle**
Email activation running in test environment. Registering as a user sends a validation email with a link that updates user status to active.

### CODE REVIEW

File: app/mailers/validation_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class ValidationMailer < ApplicationMailer
+  def inform(info, recipient)
+    # require "pry"; binding.pry
```
Comment (madhalle) [inline app/mailers/validation_mailer.rb:3]: will remove comments upon revisiting after deploying to heroku

File: app/views/validation_mailer/inform.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h2>Hello <%=@user%>!</h2>
+<p>Click <%= link_to "here", "/users/update", method: :post %> to validate your account</p>
```
Comment (madhalle) [inline app/views/validation_mailer/inform.html.erb:2]: path might need a root_path attached. Working through how default url in environment is working

File: spec/mailers/validation_spec.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe ValidationMailer, type: :mailer do
```
Comment (jpc20) [inline spec/mailers/validation_spec.rb:3]: Nice, looks good. I set up webmock and vcr on another branch so we will may have to add vcr to these tests after we merge.

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -14,16 +14,37 @@ def create
     if user.save
       session[:user_id] = user.id
       redirect_to dashboard_path
+      flash[:notice] = "Logged in as #{user.first_name}"
+      flash[:notice2] = "This account has not yet been activated. Please check your email."
+      generate_validation_email
```
Comment (jpc20) [inline app/controllers/users_controller.rb:19]: Looks Good! excited to see this in production.

File: spec/fixtures/validation_mailer/validation_email_spec.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
+
+require 'rails_helper'
```
Comment (madhalle) [inline spec/fixtures/validation_mailer/validation_email_spec.rb:2]: still working through testing properly. documentation has been proving hard to track down

**Pull request merged**